qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -753,7 +697,7 @@ def acquire(self, mode=None, samples_per_record=None,
         bytes_per_record = bytes_per_sample * samples_per_record
 
         # channels
-        channels_binrep = self._get_raw_or_bytes(self.channel_selection)
+        channels_binrep = self.channel_selection.raw_value
         number_of_channels = self.get_num_channels(channels_binrep)
```

Comment:
There is some repeating code above and below here. Maybe we can fix that in the same round?

Reply:
Sure

Reply:
Thanks!